### PR TITLE
Brandon branch dev guide & PPP

### PR DESCRIPTION
**Pull request by brandonywl**
Added 3 diagrams for the dev guide related to timetabling and 2 commands.

Also updated my PPP to include these.

Addressed #145, #171

### CODE REVIEW

**Comment by brandonywl** [comment]
Will update Timetable class diagram to include leowxiang comments on using appendix to hide the large amount of commands, allowing the class diagram to be viewable when I have time.

File: docs/team/brandon.md [hunk 1]
```diff
@@ -8,10 +8,22 @@ title : Brandon - Project Portfolio Page
...
+1. Maintain portions on commands related to Events.
+    - `add-e`
+    - `edit-e`
+    - `list-e`
+    - `delete-e`
 
 ### Contributions to Developer Guide
+1. AddEventCommand Sequence Diagram
+2. RemindCommand Sequence Diagram
+3. Timetable and Event Class Diagram
```
Comment (R-Ramana) [inline docs/team/brandon.md:29]: Markdown does auto numbering :)
```suggestion
1. AddEventCommand Sequence Diagram
1. RemindCommand Sequence Diagram
1. Timetable and Event Class Diagram
```

File: docs/team/brandon.md [hunk 2]
```diff
@@ -8,10 +8,22 @@ title : Brandon - Project Portfolio Page
 The overview below follows the following standards: 
 *Roles (short for ‘in charge of role’): Description*
 
-- 
+- Development and Maintenance of Timetable Scheduling
```
Comment (R-Ramana) [inline docs/team/brandon.md:11]: Look at my PPP to see how this is done. And look at the [recommended roles & responsibilities](https://nus-cs2113-ay2021s1.github.io/website/admin/tp-expectations.html) all the way at the bottom of the link.

**Review by R-Ramana — APPROVED** [review]

**Review by Chongjx — APPROVED** [review]

File: docs/team/brandon.md (hunk 1 above)
Comment (brandonywl) [reply docs/team/brandon.md:29]: I know just habitually typed it myself. No big issue here

File: docs/team/brandon.md (hunk 2 above)
Comment (brandonywl) [reply docs/team/brandon.md:11]: Oh I didn't see this. Thanks for pointing it out!

**Pull request merged**